homeworks/src/main/java/hw40/entity/Product.java

```diff
@@ -0,0 +1,34 @@
+package hw40.entity;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.List;
+
+@NoArgsConstructor
+@Getter
+@Setter
+@EqualsAndHashCode(of = "id")
+@ToString
+@Entity
+@Table(name = "products", schema = "postgres")
+public class Product {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(name = "name", nullable = false)
+    private String name;
+
+    @Column(name = "cost")
+    private Double cost;
+
+    @OneToMany(cascade = CascadeType.ALL, fetch = FetchType.LAZY, mappedBy = "product")
```

Comment:
remove order_product table and establish proper relationship between order and product